rerun_cpp/src/rerun/components/tensor_data_ext.cpp

```diff
@@ -0,0 +1,31 @@
+#include "tensor_data.hpp"
+
+// Uncomment for better auto-complete while editing the extension.
+// #define EDIT_EXTENSION
+
+namespace rerun {
+    namespace components {
+
+#ifdef EDIT_EXTENSION
+        struct TensorDataExt {
+#define TensorData TensorDataExt
```

Comment:
Discussion topic: Maybe we should stop with those "emulated working enviroments" using defines and structs. I started putting them here to make the _code compile_ if EDIT_EXTENSION is enabled _and_ the code wasn't generated yet. But everyone else (well, also sometimes myself!) editing these files gets this wrong and copies the pattern without copying the part that actually make it compile :/
The only _actually_ needed part is the comment markers and some means to make the code inside the comment marker not part of the sdk compilation (which is done with #ifdef EDIT_EXTENSION)

Reply:
Yeah, I'm just cargo-culting this. I'll leave it to you to clean up I think.

Reply:
😄 . Feel free to keep this unchanged in this pr!